static/src/stylesheets/module/email-signup/_form.scss

```
@@ -196,7 +195,7 @@
 // FOOTER MODS
 .footer__email-container .email-sub__iframe {
     max-width: gs-span(4);
-    min-height: 90px;
+    min-height: 9rem;
```

I _think_ dotcom is using `px` everywhere, and they have a build step to convert to `rem`s, or whatever unit we should use nowadays!

---

```suggestion
    min-height: 150px;
```
I worked out from the original of `5.625rem`  that `9rem` should be `144px`, which I've rounded up to `150px` for safety.